doc/visual_testing/reporter_visual_inspection_suite.py

```diff
@@ -34,6 +39,7 @@
 
 # %%
 # Adapted from examples/04_glm_first_level/plot_adhd_dmn.py
+@lru_cache
 def report_nifti_sphere_masker():
```

Comment:
the parameters, such as tr are tunes to a given dataset, which does not seem consistent with the function name.

Reply:
Yeah I had extracted the function out of the rest of the example code where it came from so it makes less sense that way: will inline it back.